infrastructure/clear-all-data.sh

```diff
@@ -138,17 +140,28 @@ docker run --rm --network=$NETWORK  \
   -e POSTGRES_DB="${POSTGRES_DB}" \
   -e EVENTS_MIGRATOR_ROLE="${EVENTS_MIGRATOR_ROLE}" \
   -e EVENTS_APP_ROLE="${EVENTS_APP_ROLE}" \
+  -e ANALYTICS_POSTGRES_USER="${ANALYTICS_POSTGRES_USER}" \
```

Comment:
The e2e repo needs any changes regarding this? I mean this will be different for each feature environment right?